File: ProyectoCsharp/Modelo/AdministradorDeSistema.cs


namespace Modelo
{
    public class AdministradorDeSistema : Usuario
    {
        //Constructor
        public AdministradorDeSistema()
        {
            //asdfghjk
        }

        public AdministradorDeSistema(long id_usuario, string nombre, string apellidos, string correo, string dni, string username,
                        string contraseña, string descripcion_permisos) : base(id_usuario, nombre,
                        apellidos, correo, dni, username, contraseña, descripcion_permisos)
        {

        }
    }
}

File: TrabajoJavaLp2/src/modelo/AdministradorDeSistema.java
package modelo;
public class AdministradorDeSistema extends Usuario{
	public AdministradorDeSistema(){
		
	}
	public AdministradorDeSistema(long id_usuario, String nombre, String apellidos, String correo, String codigo,
                    String username, String contrasena, String descripcion_permisos)
    {
		super(id_usuario, nombre,
                    apellidos, correo, codigo, username, contrasena, descripcion_permisos);
    }
}